golang/Release/metadata.yml

```diff
@@ -0,0 +1,6 @@
+suggested: false
+tech-preview: false
```

Comment:
I guess it should be tech-preview first?

Reply:
Not really. The tech-preview flag is used along with the suggested flag to show the blue box arount the pipeline in the UI.

See https://github.com/fabric8-launcher/ngx-launcher/blob/36c134f3e4e0ed30a76dcb954f7a3e94af35926f/projects/ngx-launcher/src/lib/components/release-strategy-step/release-strategy-step.component.html#L26-L27

Reply:
thanks!